### PR TITLE
[BIOMAGE-1041] The UI still shows the old experiment when a user launches a new experiment

### PR DESCRIPTION
**Pull request by StefanBabukov**
# Background
#### Link to issue 
https://biomage.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=BIOMAGE&modal=detail&selectedIssue=BIOMAGE-1041
#### Link to staging deployment URL 
https://ui-stefan-ui301.scp-staging.biomage.net/
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/components/data-management/ProjectDetails.jsx [hunk 1]
```diff
@@ -602,7 +604,10 @@ const ProjectDetails = ({ width, height }) => {
     }
   };
 
-  const launchAnalysis = (experimentId) => {
+  const launchAnalysis = async (experimentId) => {
+    const response = await fetchAPI(`/v1/experiments/${experimentId}`);
```
Comment (StefanBabukov) [inline src/components/data-management/ProjectDetails.jsx]: before this was done by server-side-rendering in _app, the problem is that `getInitialProps` won't be triggered when you click "launch analysis" which is why getting the info is done here with the rest of the things. Any better way to do this or this is ok?

Comment (marcellp) [reply src/components/data-management/ProjectDetails.jsx]: I believe that experiment data should be available in the Redux store somewhere. That is probably a better way to do this.

Comment (marcellp) [reply src/components/data-management/ProjectDetails.jsx]: On a separate note, why do you need to `updateExperimentInfo`?

Comment (StefanBabukov) [reply src/components/data-management/ProjectDetails.jsx]: updateExperimentInfo to put the info in the store including the analysis name

**Review by ogibson — COMMENTED** [review]
I reviewed this in the staging link and seems that the original issue was fixed. Will still need a comment in terms of the actual code changes.

File: src/components/data-management/ProjectDetails.jsx (hunk 1 above)
Comment (StefanBabukov) [reply src/components/data-management/ProjectDetails.jsx]: I see what you mean I'll rework it - the data is also under `experiments` so we can use it from there, its just that `experimentSettings/info` wont be populated then, not sure if we need it for somewhere else or just for the names

**Comment by marcellp** [comment]
Can you update the name of the ticket to reflect the actual issue name (i.e. `[BIOMAGE-1041] The UI still shows the old experiment when a user launches a new experiment`)?

[BIOMAGE-1041]: https://biomage.atlassian.net/browse/BIOMAGE-1041

**Review by marcellp — APPROVED** [review]
LGTM. This is caused by a bug (i.e. `experimentSettings` not being updated when navigating within a page) that needs further investigation and fixing, but this will fix the usability issue. @StefanBabukov to create a ticket for the underyling issue.

**Pull request merged**